components/profile/OrderByDropdown.vue

```diff
@@ -127,6 +142,7 @@ onMounted(() => {
       selectedSort.value = [sort] as string[]
     }
   }
+  calcDropdownPosition()
```

Comment:
What if the screen size changes? consider using `useEventListener(window, 'resize', fn)`